docs/src/components/SandPack/SandPack.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import React from 'react';
+import { Sandpack, SandpackProviderProps } from '@codesandbox/sandpack-react';
+
+const SandPack = (props: SandpackProviderProps) => {
+  const { template, options, files } = props;
+
+  const templateDependency = (_template: string) => {
+    switch (_template) {
+      case 'react':
+      case 'react-ts':
+        return { '@novu/notification-center': 'latest' };
+      case 'vue':
+      case 'vue-ts':
+        return { '@novu/notification-center-vue': 'latest' };
+      case 'angular':
+        return { '@novu/notification-center-angular': 'latest' };
+      default:
+        return {};
+    }
+  };
+
+  const templateDevDependency = (_template: string) => {
+    switch (_template) {
+      case 'angular':
+        return { '@types/react': 'latest' };
+      default:
+        return {};
+    }
+  };
+
+  return (
+    <Sandpack
+      theme={'dark'}
+      template={template}
+      files={files}
+      options={{
+        wrapContent: true,
+        editorHeight: 600,
+        autoReload: true,
+        autorun: false,
```

Comment:
`autorun: false` will give a run option in  in editor

<img width="399" alt="Screenshot 2023-05-12 at 10 55 22 PM" src="https://github.com/novuhq/novu/assets/39362422/3d933bdc-7f08-4de5-8062-f664cd53c291">

`autorun: true` will automatic run when this component will come in view.

I kept it false to save memory usage. If user wants to see live demo, only then it will run.